packages/splits-sdk/src/subgraph/index.ts

```diff
@@ -287,7 +287,7 @@ export const formatGqlAccount: (arg0: GqlAccount) => IAccountType = (
   }
 }
 
-const SPLITS_GRAPHQL_URL = 'http://api.splits.org/graphql'
+const SPLITS_GRAPHQL_URL = 'https://api.splits.org/graphql'
```

Comment:
ouch

Reply:
hah, i think cloudfront is setup to forward any http requests to https, so it doesn't actually matter